common/dpcpp/dpcpp_common.hpp

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "common/options.hpp"
+#include "../options.hpp"
```

Comment:
I really dislike relative paths in includes because it becomes a nightmare over time with different configurations/installations, can we avoid it here?